src/Components/Router.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+import App from '../App';
+import Repo from './Repo';
+
+const Router = () => (
+  <BrowserRouter>
+    <Route path="/" component={App} />
+    <Route path="/repo/:author/:name" component={Repo} />
+  </BrowserRouter>
```

Comment:
If you want to show different pages, you need to use `Switch` from react router.
{/* A <Switch> looks through its children <Route>s and
            renders the first one that matches the current URL. */}
You also need to add 'exact' to the first Route, the best explanation you can find there: 
https://stackoverflow.com/questions/49162311/react-difference-between-route-exact-path-and-route-path

The code should look like this:
`const Router = () => (
  <BrowserRouter>
    <Switch>
      <Route path="/" exact component={App} />
      <Route path="/repo/:author/:name" component={Repo} />
    </Switch>
  </BrowserRouter>
);
`